src/quo/components/wallet/transaction_progress/schema.cljs

```diff
@@ -0,0 +1,25 @@
+(ns quo.components.wallet.transaction-progress.schema)
+
+(def ^:private ?network
+  [:map
+   [:network {:optional true} [:maybe [:enum :mainnet :optimism :arbitrum]]]
+   [:state {:optional true} [:maybe [:enum :pending :sending :confirmed :finalising :finalized :error]]]
+   [:counter {:optional true} [:maybe :int]]
+   [:total-box {:optional true} [:maybe :int]]
+   [:epoch-number {:optional true} [:maybe :string]]
+   [:progress {:optional true} [:maybe :int]]])
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:theme :schema.common/theme]
+      [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
+      [:title {:optional true} [:maybe :string]]
+      [:tag-name {:optional true} [:maybe :string]]
+      [:tag-number {:optional true} [:maybe [:or :string :int]]]
+      [:tag-photo {:optional true} [:maybe [:or :schema.common/image-source :string]]]
```

Comment:
Another example of `or :string`. I strongly feel we should update `shema.common/image-source` to accept strings.